gui/src/2D/controls/image.ts

```diff
@@ -312,7 +312,7 @@ export class Image extends Control {
         const width = this._domImage.width;
         const height = this._domImage.height;
 
-        // Shoudl abstract platform instead of using LastCreatedEngine
+        // Should abstract platform instead of using LastCreatedEngine
         const engine = Engine.LastCreatedEngine;

```

Comment:
I wonder if it should be along those lines: 

this._host?.getScene()?.getEngine() || Engine.LastCreatedEngine; ?